contracts/common/mixin/RootChainable.sol

```diff
@@ -1,14 +1,13 @@
-pragma solidity ^0.4.24;
+pragma solidity ^0.5.2;
```

Comment:
I don't think we need this file anymore.

Reply:
let's see for now including it here, I am kinda rethinking some wiring so will remove if not useful.

Reply:
Curious to know what you have in mind.